includes/http/WP_AI_Client_Client_Adapter.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * WordPress AI Client HTTP Client Adapter
+ *
+ * @package WordPress\AI_Client
+ * @since n.e.x.t
+ */
+
+namespace WordPress\AI_Client\HTTP;
+
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ResponseFactoryInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+
+/**
+ * PSR-18 HTTP Client adapter using WordPress HTTP API
+ *
+ * This adapter allows WordPress HTTP functions to be used
+ * as a PSR-18 compliant HTTP client.
+ *
+ * @since n.e.x.t
+ */
+class WP_AI_Client_Client_Adapter implements ClientInterface {
```

Comment:
I would simply call this class something like `WordPress_HTTP_Client`:
* While it adapts the WordPress HTTP functions to work with PSR, itself it's simply a PSR client, also per the interface it implements.
* It sends HTTP requests, so it has nothing to do with AI directly :)

Reply:
Agreed! Resolved in 3998eb7